CHANGELOG.md

```diff
@@ -7,8 +7,14 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+### Added
+- Layer integration tests for more apps. Closes
+  [[#472](https://github.com/metalbear-co/mirrord/issues/472)].
```

Comment:
This PR adds a test with go 1.19.0. Do we also need one with go 1.18?
In that case I have to change this:
```suggestion
- Layer integration tests for more apps. Addresses but does not close
  [[#472](https://github.com/metalbear-co/mirrord/issues/472)] (go 1.18 still missing).
```